media/server/gstplayer/source/GstGenericPlayer.cpp

```diff
@@ -498,6 +498,62 @@ GstElement *GstGenericPlayer::getDecoder(const MediaSourceType &mediaSourceType)
     return nullptr;
 }
 
+GstElement *GstGenericPlayer::getFilter(const MediaSourceType &mediaSourceType)
```

Comment:
This functions gets parser, not filter (from gstreamer type classification perspective). 
Have you considered using gstElementFactoryListGetElements? That will give you a list of of factories that match the given type. Then, you would be able to iterate through them to find the property.